indigo_app/static/javascript/indigo/views/document_editor.js

```diff
@@ -148,12 +156,21 @@
       this.$('.document-sheet-container').scrollTop(0);
     },
 
+    addNewEditTime: function(e) {
```

Comment:
I'd suggest three methods which would better describe what's going on here:

* `editActivityStarted`: set `editStartedAt`
* `editActivityEnded`: stash a new edit activity item and clear `editStartedAt`
* `editActivityCancelled`: clear `editStartedAt`

Reply:
It would mean editStartedAt is set and cleared only by those methods, rather than in a mix of other methods. I think it would be clearer that the three methods work in tandem with each other, and make this piece of functionality easier to reason about.

Reply:
done